.github/jobscripts/detect_dump.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -eux
```

Comment:
And consider adding `set -o pipefail`

Reply:
I don't see any downside from switching to bash.

While there is (limited) value in staying POSIX compliant for the scripts that come with DOMjudge, I don't see the value for github actions.